app/util.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const request = require('request-promise');
+
+// returns a Date representing tomorrow at 12pm so that
+// we can issue darksky 'time machine' requests to get an
+// idea of the weather tomorrow at a time that should be
+// relevant to someone interested in the weather
+function getTomorrowAtNoon() {
+    let tomorrow = new Date();
+    tomorrow.setDate(tomorrow.getDate() + 1);
+    tomorrow.setHours(12);
+    return tomorrow;
+}
+
+// fixed coordinates used to query darksky
+const latitude = 38.8892681;
+const longitude = -77.0501425;
+const darkskyToken = process.env.DARKSKY_TOKEN || '';
+const darkskyUrlBase = `https://api.darksky.net/forecast/${darkskyToken}/${latitude},${longitude}`;
+
+// returns a promise that resolves to the JSON representing the current weather
+// report from darksky on success
+function getCurrentWeatherReport() {
+    // we don't need the minutely or hourly forecast, so we'll exclude them
+    const darkskyCurrentlyUrl = `${darkskyUrlBase}` + '?exclude=minutely,hourly';
+    return request({ uri: darkskyCurrentlyUrl, json: true });
+}
+
+function getTomorrowsWeatherReport() {
+    const darkskyTomorrowUrl = `${darkskyUrlBase}` + getTomorrowAtNoon().getTime() + '?exclude=currently,minutely,hourly';
+    return request({ uri: darkskyTomorrowUrl, json: true });
+}
+
+function generateCurrentSummary(dataBlock) {
+    return generateWeatherReport(dataBlock, 'is currently', 'feels like');
```

Comment:
nit: these string variables are easily confusable. i'd probably change the method signature to expect a single object with named 'variables'